src/components/message-parts/link-attachments/parts/embedActionsTheme.js

```diff
@@ -17,15 +17,23 @@ export const styles = {
     fontSize: 0,
     lineHeight: 0,
     userSelect: 'none',
-    marginLeft: 10,
-    '&:first-child': {
-      marginLeft: 0,
-    },
     opacity: 0.7,
+    '&:not(:first-child)': {
+      // This duplicate needs to prevent a bug in JSS
```

Comment:
Which bug is it? Can you link to an issue or describe it. This way we can clean up once the bug is fixed.